engines/sky/detection.cpp

```diff
@@ -66,6 +66,10 @@ class SkyMetaEngineDetection : public MetaEngineDetection {
 	PlainGameList getSupportedGames() const override;
 	PlainGameDescriptor findGame(const char *gameid) const override;
 	DetectedGames detectGames(const Common::FSList &fslist, uint32 /*skipADFlags*/, bool /*skipIncomplete*/) override;
+
+	uint getMD5Bytes() const override {
+		return 0;
```

Comment:
Missing a TODO here

Reply:
I added the TODO to the GLK engine since that uses MD5s for detection. The Sky engine doesn't use MD5s, so there's no need to implement anything here.